plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeConfig.java

```diff
@@ -308,6 +309,18 @@ public DeltaLakeConfig setCheckpointRowStatisticsWritingEnabled(boolean checkpoi
         return this;
     }
 
+    public boolean isCheckpointPartitionFilterEnabled()
```

Comment:
keep accessors in same order as the fields (eg add after defaultCheckpointWritingInterval everywhere)